docs/en/writing-running-appium/tutorial/swipe/simple-screen.md

```diff
@@ -0,0 +1,70 @@
+## Simple Screen swipe
+
+Phone applications have interface. Interface, in most cases, blocks your
+swipe actions when you start from it. Historically it uses header and/or
+footer. To make screen swipe more reliable - it is better to start swipe
+action from center of screen.
+
+![swipe_screen](simple-screen.png)
+
+```java
+/**
+ * Performs screen swipe from center
+ *
+ * @param dir d - down, u - up, l - left, r - right
+ * @version java-client: 7.3.0
+ **/
+public void swipeScreen(String dir) {
+    System.out.println("swipeScreen(): dir: '" + dir + "'"); // always log your actions
+
+    int screenHeight, screenWidth;
+    int edgeBorder = 10; // better avoid edges
```

Comment:
I'd rather provide this value in percents

Reply:
just no reason. it different for width and height. hard to decide and add more lines of code that hardly will bring any real value.